layouts/partials/footer.html

```diff
@@ -1,15 +1,13 @@
-{{ $logo := printf "img/logos/%s" site.Params.logos.footer | relURL }}
```

Comment:
This is misleading in its complexity: use of `relURL` doesn't actually do anything worthwhile here, given that logos are globally accessible resources.